lib/page/reset_password/reset_password_page.dart

```diff
@@ -1,11 +1,71 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:survey/gen/assets.gen.dart';
+import 'package:survey/resource/dimens.dart';
+import 'package:survey/widget/app_bar_back_button_widget.dart';
+import 'package:survey/widget/onboarding_background_widget.dart';
+import 'package:survey/widget/rounded_button_widget.dart';
+import 'package:survey/widget/text_input_widget.dart';
 
 class ResetPasswordPage extends StatelessWidget {
   const ResetPasswordPage({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    // TODO: Implement Reset Password page
-    return Container();
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Stack(
+        children: [
+          OnboardingBackgroundWidget(
+            background: AssetImage(Assets.images.bgOnboarding.path),
+            shouldBlur: true,
```

Comment:
Is there any case `shouldBlur` accept a false value? Do we need this param?

Reply:
I plan to use this widget for the background image of the Home screen, the Questions screen, and the Survey Detail screen also. Please check out https://github.com/chornerman/Survey/pull/88 for the complete implementation 😄